src/components/RadioButtonGroup/styled/marker.js

```diff
@@ -3,14 +3,15 @@ import styled from 'styled-components';
 import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
 import { BORDER_RADIUS_2 } from '../../../styles/borderRadius';
 
+const sizeMap = { large: '3.1rem', medium: '2.6rem', small: '1.76rem', 'x-small': '1.26rem' };
 const StyledMarker = attachThemeAttrs(styled.span)`
     position: absolute;
     background: ${props => props.palette.background.main};
     opacity: 0;
     top: 0;
     bottom: 0;
     margin: auto;
-    height: 2.6rem;
+    height: ${props => sizeMap[props.size] || sizeMap.medium};
```

Comment:
@TahimiLeonBravo will be able to change to 2.5rem here?

Reply:
@rgah2107 @yvmunayev `2.6rem` is right because we need to make the marker a little higher than the container.
Also, we need to change by `small='1.8rem'` and ` 'x-small': '1.3rem'` because right now this is not centered vertically
![Screen Shot 2020-05-14 at 3 41 41 PM](https://user-images.githubusercontent.com/32083556/81986946-665faf00-95fe-11ea-9a18-5b71d582b746.png)
![Screen Shot 2020-05-14 at 3 41 59 PM](https://user-images.githubusercontent.com/32083556/81986950-6790dc00-95fe-11ea-9c1b-192d36725508.png)